package-system/sdformat/build_config.json

```diff
@@ -6,6 +6,7 @@
    "package_url":"https://github.com/gazebosim/sdformat.git",
    "package_license":"MIT",
```

Comment:
Isn't this Apache licensed?

Reply:
You are correct, it is under Apache 2.0 license.
I'll fix it.